HoloLensCommander/HoloLensCommander/Utilities/UserToken.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HoloLensCommander
+{
+    public class UserToken
```

Comment:
Do you think we will need anything other than the token string? 
Is the token string in a special format that should be validated?
If not, I would recommend removing the UserToken class and just use the string throughout.

Reply:
@davidkline-ms No, so far I have only needed the token. I am also working to add username and password support, but am using a NetworkCredentials object for that.

Reply:
maybe encapsulate everything here (network credential / token string)? i definitely recommend requiring the user to enter credentials at least on first use during a session. these credentials would pose a security risk. 

As an asside, i am considering removing the device portal credential storage in a future release

Reply:
Yea, I agree it should definitely not be stored as plain text. What about storing it as a secure string? What kind of credential manager is built in to UWP apps?

Reply:
SecureString may work, but I haven't tried storing it in a config file. Havent researched credential manangement in UWP apps.

Reply:
https://docs.microsoft.com/en-us/appcenter/api-docs/ This is what we're attempting to use. App Center recommends issuing a token to make API calls. I can't find any docs on app center for how to use OAuth. Do you know how?